CHANGES.rst

```diff
@@ -24,6 +24,12 @@
 Changes
 =======
 
+Version 1.0.0a12 (released 2016-06-29)
```

Comment:
This was not necessary, since we agreed that we don't do any release notes for alpha releases... Besides the canonical way of preparing release notes:

```
$ kwalitee prepare release v1.0.0a11..v1.0.0a12
```

does not announces anything.